src/main/scala/outwatch/dom/package.scala

```diff
@@ -25,8 +25,8 @@ package object dom extends Attributes with Tags with HandlerFactories {
 
   implicit def compositeModifier(modifiers: Seq[VDomModifier]): VDomModifier = IO.pure(CompositeVDomModifier(modifiers))
 
-  implicit class ioVTreeMerge(vnode: VNode) {
-    def apply(args: VDomModifier*): VNode = {
+  implicit class ioVTreeMerge(vnode: IO[VTree]) {
```

Comment:
Should we introduce `type VTree = IO[VTree_]`, as we did for VNode and VDomModifier? We will need to use this type for the apply method.

Reply:
I updated the PR so that the signature of `ioVTreeMerge` remains unchanged. If we can avoid it, I think it's better to not expose `VTree`, the fewer types the users have to deal with the better.